site/en/guides/thread-primer/ipv6-addressing.md

```diff
@@ -278,9 +278,159 @@ What you've learned:
 To learn more about Thread's IPv6 addressing, see sections 5.2 and 5.3 of the
 [Thread Specification](http://threadgroup.org/ThreadSpec).
 
+
+## Check your understanding
+
+<div>
+  <devsite-multiple-choice>
+    <div>A Thread network consists of which three scopes?</div>
```

Comment:
Thread can support the full set of IPv6 multicast scopes as defined in [RFC 7346](https://datatracker.ietf.org/doc/html/rfc7346).

We may want to reformulate this question to test knowledge of how some of the scopes map onto Thread networks.